mm2src/coins/lp_coins.rs

```diff
@@ -1259,6 +1259,8 @@ pub struct SendTakerFundingArgs<'a> {
     pub maker_pub: &'a [u8],
     /// DEX fee
     pub dex_fee: &'a DexFee,
+    /// The extra fee added to the trading volume to cover the taker payment transaction fees
+    pub taker_payment_fee: BigDecimal,
```

Comment:
tbh the namings got my mind pretty entangled for a bit.

I agree `funding_spend_fee` is a better name. but i changed it to `taker_payment_fee` since it makes the names more symmetrical with the maker side.

`funding_fee` - `taker_payment_fee` - `taker_payment_spend_fee`
`-----------` - `maker_payment_fee` - `maker_payment_spend_fee`

could adding `_tx` make it less confusing here? like `taker_payment_tx_fee`? otherwise I would suggest sticking to `funding_spend` everywhere else (not only the fees).

Reply:
> could adding _tx make it less confusing here? like taker_payment_tx_fee?

`taker_payment_fee` is fine, it's not a big deal.